src/Sign.Cli/CodeCommand.cs

```diff
@@ -114,5 +115,20 @@ private static HashAlgorithmName ParseHashAlgorithmName(ArgumentResult result)
                     return HashAlgorithmName.SHA256;
             }
         }
+
+        private static Uri? ParseUrl(ArgumentResult result)
+        {
+            if (result.Tokens.Count != 1 ||
+                !Uri.TryCreate(result.Tokens[0].Value, UriKind.Absolute, out Uri? uri) ||
+                !(string.Equals(Uri.UriSchemeHttp, uri.Scheme, StringComparison.OrdinalIgnoreCase) ||
+                    string.Equals(Uri.UriSchemeHttps, uri.Scheme, StringComparison.OrdinalIgnoreCase)))
+            {
+                result.ErrorMessage = $"Unsupported value for --{result.Argument.Name}.  The value must be an HTTP or HTTPS URL.";
```

Comment:
Do we need to localize this string? It looks user facing info.

Reply:
Is there test for this one?  We're mutating the original input to set error and same time returning `null`, not sure if it's working as expected unless there is actual test for this.

Reply:
Great catch!  I missed this string because of the order in which I applied (cherry-picked) a commit.  I fixed it in the other PR which is focused on localizability.  https://github.com/dotnet/sign/pull/543/commits/d72c060ed6a7604fac16e491f0955d308ea3d9eb

Reply:
As far as tests go, yes, there are positive and negative tests [here](https://github.com/dotnet/sign/pull/540/files#diff-1a2ec79601f9b01f7dc6a290224d0c978bc8691aa175c4063d19b1dd0336bd1dR24-R41).  (The tests are shared by all `Uri`-based option tests.)

Setting the `ErrorMessage` property to a non-`null` value generates a parse error and prevents invocation, so a `null` `Uri` will not happen in positive cases.  See System.CommandLine [documentation](https://learn.microsoft.com/en-us/dotnet/api/system.commandline.parsing.symbolresult.errormessage?view=system-commandline#remarks).

The comment [here](https://github.com/dotnet/sign/pull/540/files#diff-bc5f203d5b2bd02ed1b817ea63cc6ab928a45e92dd1b0a259f948c2beb0c1f2dR65-R68), where the value is actually used, also covers this.